README.md

```diff
@@ -76,6 +76,163 @@ For more info about the API, see the [API reference](https://kontent.ai/learn/re
 
 You can find the Delivery and other SDKs at <https://github.com/Kentico>.
 
+## About
+
+This section describes the application.
+
+### Used toolchain
+
+This application is based on the [Create React App](https://reactjs.org/docs/create-a-new-react-app.html) using the following template `--template typescript`.
+
+### Model mapping and data fetching
+
+There are two types of model mapping in this application:
+
+#### content type -> DTO -> component
+
+For generating component models from content types, we have used [Kontent.ai model generator](https://github.com/Kentico/kontent-model-generator-js) tool. All generated models can be found in `src/Models` folder. The `_project.ts` exports `projectModel` which contains information about the project structure such as project languages as well as other information such as codenames about content types. Generated models are used to obtain correctly typed objects via client.

```

Comment:
```suggestion
Content type definitions are being generated from content types via [Kontent.ai model generator](https://github.com/Kentico/kontent-model-generator-js) tool. All generated types can be found in `src/Models` folder. The `_project.ts` contains information about the project structure such as project languages as well as other structure information such as codenames about content types.
```

We might want to adjust the naming - it is not a model anymore - it is a type... soryy I might have point that out sooner.